### PR TITLE
[SearXNG] move from searx/searx-instances to searxng/searx-instances

### PR DESCRIPTION
**Pull request by return42**
Closes: https://github.com/searxng/searx-instances/issues/3

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,8 +1,8 @@
-## For searx instance operators
+## For SearXNG instance operators
 
-* [Create an issue](https://github.com/searx/searx-instances/issues/new/choose) to add / remove / edit a searx instance on https://searx.space/
-* If you add a new instance, allow the IPs of ```check.searx.space``` to access your instance. It will checks your instance periodically ([source code](https://github.com/searx/searx-stats2)). The results are displayed on https://searx.space/
-* Wait for a reviewer to actually change [searxstats/instances.yml](https://github.com/searx/searx-instances/blob/master/searxinstances/instances.yml)
+* [Create an issue](https://github.com/searxng/searx-instances/issues/new/choose) to add / remove / edit a searx instance on https://searx.space/
+* If you add a new instance, allow the IPs of ```check.searx.space``` to access your instance. It will checks your instance periodically ([source code](https://github.com/searxng/searx-stats2)). The results are displayed on https://searx.space/
```
Comment (dalf) [inline README.md:4]: The repo name for searx.space is `https://github.com/searxng/searx-space`

File: .github/ISSUE_TEMPLATE/add-instance.yaml [hunk 2]
```diff
@@ -48,7 +48,7 @@ body:
       label: Source code URL
       description: |
         If you have modified the source code, please publish it to an URL and make sure that the URL is publicly accessible in your instance's site
-        ( Check your [settings.yml](https://github.com/searx/searx/blob/e88bea53ebf220648685cdb8eff017357728494e/searx/settings.yml#L8) )
+        ( Check your [buildenv](https://docs.searxng.org/dev/makefile.html#make-buildenv) )
```
Comment (dalf) [inline .github/ISSUE_TEMPLATE/add-instance.yaml:51]: It won't work for docker installation.

File: README.md (hunk 1 above)
Comment (return42) [reply README.md]: thanks / fixed

File: .github/ISSUE_TEMPLATE/add-instance.yaml (hunk 2 above)
Comment (return42) [reply .github/ISSUE_TEMPLATE/add-instance.yaml:51]: :-) .. this was the point where I opened https://github.com/searxng/searxng/issues/728

Please push your suggestion on top of this branch / Sorry, I do not have more time today.

Comment (dalf) [reply .github/ISSUE_TEMPLATE/add-instance.yaml:51]: I don't understand how it is related and how it helps for the docker installation.

There is no usable `manage` script with the docker image, only `settings.yml`.

Comment (dalf) [reply .github/ISSUE_TEMPLATE/add-instance.yaml]: See c102320 : the comment asks to check git_url for searx instance (SearXNG relies on the git command to get the GIT URL, there is nothing to do). We can update this field later.

**Comment by dalf** [comment]
I'm going to merge this PR since the change in `searxinstances/update.py` is required to deal with the other opened issues.

File: .github/ISSUE_TEMPLATE/add-instance.yaml (hunk 2 above)
Comment (return42) [reply .github/ISSUE_TEMPLATE/add-instance.yaml]: > there is nothing to do

Thanks, c102320 was exactly what I asked for.

> I'm going to merge this PR 

Perfect .. this was what I expected :-)

**Pull request merged**